extensions/jaxb/deployment/src/main/java/io/quarkus/jaxb/deployment/JaxbClassesToBeBoundBuildItem.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.jaxb.deployment;
+
+import java.util.List;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * List of classes to be bound in the JAXB context.
+ */
+public final class JaxbClassesToBeBoundBuildItem extends MultiBuildItem {
+
+    private final List<Class<?>> classes;
```

Comment:
Maybe store only the class names here (`List<String>`). Other extensions may have use cases where the `Class` isn't available

Reply:
I didn't want to do this at first because we will be loading the classes from the class names at runtime (so, the startup time will be degraded). But perhaps it will be insignificant and using a list of strings is more usable. 
So, changed following your suggestion.

Reply:
Yes, I agree that it should be insignificant (Depending on the number of classes though), but you can avoid reflection completely if you generate the producer class in a build step